Makefile

```diff
@@ -49,7 +49,7 @@ test: $(BIN)
 	@result=0; \
 	for MOD in $(GOMODULES); do \
 		cd $${MOD}; \
-		go test -v ./... || result=1; \
+		go test -v -timeout 90s ./... || result=1; \
```

Comment:
the reason for this change is that the filtering scenario takes quite some amount of time when executed as table tests. This will be adapted accordingly, once we start to implement real testing scenarios for filtering